### DISCUSSION TITLE
GRASS error

### DISCUSSION THREAD

**Discussion by valterhydrodynamics** [OSGeo/grass#2253] [Q&A]
Morning,
GRASS GIS ("Long term" and 8.0.1") does not start up. Gives the following message:

//
Launching <wxpython> GUI in the background, please wait...
Microsoft Windows [Version 10.0.19042.1526]
(c) Microsoft Corporation. Todos os direitos reservados.

C:\Users\Particular\Documents>Traceback (most recent call last):
  File "C:\OSGeo4W\apps\Python39\lib\site-packages\wx\core.py", line 3407, in <lambda>
    lambda event: event.callable(*event.args, **event.kw) )
  File "C:\OSGeo4W\apps\grass\grass80\gui\wxpython\wxgui.py", line 88, in show_main_gui
    mainframe = GMFrame(parent=None, id=wx.ID_ANY, workspace=self.workspaceFile)
  File "C:\OSGeo4W\apps\grass\grass80\gui\wxpython\lmgr\frame.py", line 156, in __init__
    self._createConsole(self.notebook)
  File "C:\OSGeo4W\apps\grass\grass80\gui\wxpython\lmgr\frame.py", line 379, in _createConsole
    self.goutput = GConsoleWindow(
  File "C:\OSGeo4W\apps\grass\grass80\gui\wxpython\gui_core\goutput.py", line 127, in __init__
    self.cmdPrompt = GPromptSTC(
  File "C:\OSGeo4W\apps\grass\grass80\gui\wxpython\gui_core\prompt.py", line 155, in __init__
    GPrompt.__init__(self, parent=parent, giface=giface, menuModel=menuModel)
  File "C:\OSGeo4W\apps\grass\grass80\gui\wxpython\gui_core\prompt.py", line 57, in __init__
    self.mapList = self._getListOfMaps()
  File "C:\OSGeo4W\apps\grass\grass80\gui\wxpython\gui_core\prompt.py", line 114, in _getListOfMaps
    result["raster"] = grass.list_strings("raster")
  File "C:\OSGeo4W\apps\grass\grass80\etc\python\grass\script\core.py", line 1462, in list_strings
    for line in read_command(
  File "C:\OSGeo4W\apps\grass\grass80\etc\python\grass\script\core.py", line 606, in read_command
    return handle_errors(returncode, stdout, args, kwargs)
  File "C:\OSGeo4W\apps\grass\grass80\etc\python\grass\script\core.py", line 429, in handle_errors
    raise CalledModuleError(module=module, code=code, returncode=returncode)
grass.exceptions.CalledModuleError: Module run `g.list --q -m type=raster` ended with an error.
The subprocess ended with a non-zero return code: 3221225785. See errors above the traceback or in the error output.
//
Any clue to solve it?

**Comment by ninsbl** [comment]
It may be related to your portuguese system local (`Todos os direitos reservados`), combined with encoding issues on MS Windows...

**Reply by hellik** [reply]
It looks like a dependencies issue. Do you have more sqlite in your %PATH% ?

**Reply by valterhydrodynamics** [reply]
> It may be related to your portuguese system local (`Todos os direitos reservados`), combined with encoding issues on MS Windows...

My settings have English (USA). I have never had this issue

**Reply by valterhydrodynamics** [reply]
> 
hellik, don't know how to see what you mean

**Reply by valterhydrodynamics** [reply]
> It may be related to your portuguese system local (`Todos os direitos reservados`), combined with encoding issues on MS Windows...
Formatted my laptop and still have the same issue. [Curious fact - also work with HEC-HMS and since 4.7.1 version i also have issues]

**Reply by hellik** [reply]
> > 
> 
> hellik, don't know how to see what you mean 

Some explanations: https://grasswiki.osgeo.org/wiki/WinGRASS_errors

**Reply by valterhydrodynamics** [reply]
> It looks like a dependencies issue. Do you have more sqlite in your %PATH% ?

![PATH](https://user-images.githubusercontent.com/43525314/157064478-2efd9527-c158-4d75-8ced-328600230932.PNG)


My "path" have this: https://drive.google.com/file/d/1ou9kN4FqJ8ZKFx8a8M7-A-
![PATH]

**Reply by valterhydrodynamics** [reply]
> It looks like a dependencies issue. Do you have more sqlite in your %PATH% ?
I'm doing a search to "sqlite3.dll" to show here the situation

![sqlite](https://user-images.githubusercontent.com/43525314/157055834-a4d0d146-d1e6-4480-b3f9-d9004d5e7dbf.PNG)

**Reply by hellik** [reply]
Could you rename temporarily all the other sqlite3.dll, except those of OSGeo4W, and try to start winGRASS?

Then rename step by step the others while starting winGRASS after each renaming step.

**Reply by valterhydrodynamics** [reply]
> 
![2](https://user-images.githubusercontent.com/43525314/157096952-d52a4193-24d4-44a8-820e-a19d051bdc10.PNG)
![3](https://user-images.githubusercontent.com/43525314/157097192-583dea38-aa6d-4587-99da-ee641994847a.PNG)

Changed the name but still have the same problem
Also I'm unable to to run a simple grass algorithm

See images

**Comment by ninsbl** [comment]
OK, googling your error returncode it seems it might by a missing dll:
https://stackoverflow.com/questions/18668003/the-procedure-entry-point-gxx-personality-v0-could-not-be-located
 or related to compilation.
Can you point us to the specific installer you were using so we could try to replicate?

**Reply by valterhydrodynamics** [reply]
I used the OSGeo4W network installer in: https://qgis.org/en/site/forusers/download.html#
"osgeo4w-setup.exe"

**Comment by valterhydrodynamics** [comment]
Posted there, yesterday, as a bug "classification":
https://github.com/OSGeo/grass/issues/2255#issuecomment-1061143643
Thanks
